portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -17,15 +17,31 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Gson gson = new Gson();
-    String json;
+    String jsonObj= "";
     UserService userService = UserServiceFactory.getUserService();
     if (userService.isUserLoggedIn()) {
       String userEmail = userService.getCurrentUser().getEmail();
-      json = gson.toJson(userEmail);
+      String urlToRedirectToAfterUserLogsOut = "/";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+      jsonObj = "{";
```

Comment:
(optional) Constructing your own json string is more error-prone than using gson.toJson(). Maybe considering creating a simple value class as your data structure? AutoValue is great for this https://github.com/google/auto/blob/master/value/userguide/index.md#how-to-use-autovalue, maybe give it another shot?

Reply:
would you happen to know what the error could be? (I've run it with -X and -e and neither were super helpful ahh)
![image](https://user-images.githubusercontent.com/48307028/84083838-fe577b00-a9a7-11ea-9ab8-f400906a6d9b.png)

Reply:
This is in my pom.xml file
![Screenshot 2020-06-08 at 4 56 22 PM](https://user-images.githubusercontent.com/48307028/84084410-0d8af880-a9a9-11ea-9423-2ced58d9098d.png)

Reply:
I've also tried not using the word public and moving the file into the servlets package, and it still says cannot find symbol

Reply:
just as a reference, this is what my filetree and code look like:
![image](https://user-images.githubusercontent.com/48307028/84085183-a8380700-a9aa-11ea-8eb9-1f18dd79344a.png)

Reply:
You would also need the dependency on `auto-value` (the annotation processor), https://github.com/google/auto/blob/master/value/userguide/index.md#with-maven